python/rmm/rmm.py

```diff
@@ -267,6 +267,44 @@ def get_ipc_handle(ary, stream=0):
     )
 
 
+try:
+    import cupy
+except ImportError:
```

Comment:
Could you use `Exception` due to Cython-related issue (https://github.com/cupy/cupy/issues/2148)?
```suggestion
except Exception:
```